sdk/identity/azure-identity/inc/azure/identity/detail/client_credential_core.hpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include "azure/identity/dll_import_export.hpp"
+
+#include <azure/core/credentials/credentials.hpp>
+#include <azure/core/url.hpp>
+
+#include <string>
+
+namespace Azure { namespace Identity { namespace _detail {
+  class ClientCredentialCore final {
+    Core::Url m_authorityHost;
+    std::string m_tenantId;
+    bool m_isAdfs;
```

Comment:
Curious: What's Adfs?

Reply:
Active Directory Federated Services, a tenant that is used in Azure Stack.

Reply:
nit: Adding a doc comment would be helpful for readability.